src/Store/MongoDB/Store.php

```diff
@@ -126,7 +126,7 @@ public function query(Vector $vector, array $options = []): array
 
         foreach ($results as $result) {
             $documents[] = Document::fromVector(
-                Vector::create1536($result[$this->vectorFieldName]),
```

Comment:
So the static one is still needed?

Reply:
Nope, just thought it is an unnecessary Detail and limitation here